expcore/Mini_games.lua

```diff
@@ -207,8 +263,39 @@ function Mini_games.error_in_game(error_game)
     Mini_games.stop_game()
     game.print("an error has occured things may be broken, error: "..error_game)
 end
+
+local kick_all =
+function()
+    for i,player in ipairs(game.connected_players) do
+        game.kick_player(player,"You cant stay here")
+    end
+end
+
+Commands.new_command('kill_all','Command to stop a mini_game.')
+:register(function(_,_)
+    kick_all()
+end)
+
+Commands.new_command('stop_games','Command to stop a mini_game.')
+:register(function(_,_)
+    for i,player in ipairs(game.connected_players) do
+        player.connect_to_server{address=global.servers["lobby"],name="lobby"}
+    end
+end)
+Commands.new_command('set_lobby','Command to stop a mini_game.')
```

Comment:
This one has an ok name, but description is wrong.

Reply:
see up